### ISSUE TITLE
Notification - WCAG rule 4.1.3

### ISSUE THREAD

**Issue by SeamusLeonardHPE** [grommet/grommet#7607]
**Failed rule**

4.1.3
Status Messages
Level AA

In content implemented using markup languages, status messages can be programmatically determined through role or properties such that they can be presented to the user by assistive technologies without receiving focus.

---

**Notes**

Failing since the toast notification is not being announced

This rule is especially relevant for toast and inline notifications, because they:

Often appear dynamically (e.g., after form submission)
Do not receive focus, but still need to be communicated

To comply:

Use ARIA roles like:
- role="status" → polite updates (e.g., "Changes saved")
- role="alert" → urgent updates (e.g., "Connection lost")

Ensure the content is present in the DOM and updated when the message changes
The container should not receive focus, but must be in the accessibility tree

**Comment by britt6612** [comment]
@jcfilben 

thoughts on the notes:

```

Notes

Failing since the toast notification is not being announced

This rule is especially relevant for toast and inline notifications, because they:

Often appear dynamically (e.g., after form submission)
Do not receive focus, but still need to be communicated

To comply:

Use ARIA roles like:

role="status" → polite updates (e.g., "Changes saved")
role="alert" → urgent updates (e.g., "Connection lost")
Ensure the content is present in the DOM and updated when the message changes
The container should not receive focus, but must be in the accessibility tree

```

from above but I know we talked about just only having the `role="status"` we still good with that route?

**Comment by jcfilben** [comment]
I'm good with just using role="status"

**Comment by britt6612** [comment]
closed via https://github.com/grommet/grommet/pull/7708
and updated on DS site
